### ISSUE TITLE
BLES02258 Fifa 19: Game can't be loaded or unplayable after this commit 7cf11c763796b6ccc999ea749faa0e0399ef6be1

### ISSUE THREAD

**Issue by e88z4** [RPCS3/rpcs3#5994]
I did a git bisect for an issue that I am experiencing with Fifa 19. 

The first bad commit is cc8c6358552f47627e7fda422141615880a71bd7. Before this commit, the game is playable with 30 FPS and stuttering graphic and audio. Below is the snapshot of the error of the first bad commit.  

```
F {Worker 1} SPU: Trampoline simplified at 0x19814 (level=1) x4
F {Worker 4} SPU: Trampoline simplified at 0x197e0 (level=1) x4
F {Worker 3} SPU: Trampoline simplified at 0x19d90 (level=1) x4
F {Worker 0} SPU: Trampoline simplified at 0x199b8 (level=1) x4
F {Worker 7} SPU: Trampoline simplified at 0x1af50 (level=1) x4
F {Worker 5} SPU: Trampoline simplified at 0x1aa88 (level=1) x4
F {Worker 9} SPU: Trampoline simplified at 0x1aa74 (level=1) x4
F {Worker 11} SPU: Trampoline simplified at 0x1bfa8 (level=1) x4
F {Worker 2} SPU: Trampoline simplified at 0x1a9e0 (level=1) x4
F {Worker 10} SPU: Trampoline simplified at 0x1aa64 (level=1) x4
E {PPU[0x1000000] Thread (main_thread)} SPU: SPU Runtime: Cache building failed (too much data). SPU Cache will be disabled.
U {PPU[0x1000000] Thread (main_thread) [0x0269fe64]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000000] Thread (main_thread) [0x026a00bc]} sys_prx TODO: _sys_prx_register_library(library=*0x26a26a0)
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-1z3Pyu8YVnnorJiQ2hfu1e-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libsysmodule.sprx
U {PPU[0x1000000] Thread (main_thread) [0x026e3d8c]} HLE TODO: Unimplemented syscall sys_ss_access_control_engine -> CELL_OK
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} PPU: LLVM: Loaded module v3-tane-XCgQWMc1s99n1WTwrpWvw4-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libgcm_sys.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} PPU: LLVM: Loaded module v3-tane-TP3c8YsdNggjaG5WRjcboe-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} PPU: LLVM: Loaded module v3-tane-LWAKW9c35vXJad6uHkwzhK-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libsre.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} PPU: LLVM: Loaded module v3-tane-fLV79QqHgPi90UziTsWrHo-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libfs.sprx
U {PPU[0x1000000] Thread (main_thread) [0x026a061c]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000000] Thread (main_thread) [0x026a0208]} sys_prx TODO: _sys_prx_register_library(library=*0x1e3aeec)
U {PPU[0x1000000] Thread (main_thread) [0x026a0208]} sys_prx TODO: _sys_prx_register_library(library=*0x1e3af08)
U {PPU[0x1000000] Thread (main_thread) [0x026a0208]} sys_prx TODO: _sys_prx_register_library(library=*0x1e3af24)
U {PPU[0x1000000] Thread (main_thread) [0x026a0870]} sys_prx TODO: _sys_prx_get_module_list(flags=2, pInfo=*0xd0040810)
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-2my7r7Nd2rncZdtcuT9g60-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-aaAKL9HJRRAWFrQ1t0vFNk-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libnet.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-GidnR56pzgun0Z5aqipN0L-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libusbd.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-ZAhea3FFhakpfMa36ZvaNx-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/librtc.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-TQYHfeqGWfX22Nd4Fh0ZJb-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libcelpenc.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-gchiVUbW1b4kPJPTmwJ8QL-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libadec.sprx
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} PPU: LLVM: Loaded module v3-tane-f3b6kdGAVh5RqMYJzSf3KC-000007-znver1.obj
S {PPU[0x1000000] Thread (main_thread) [0x026a09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libcelpdec.sprx
E {PPU[0x1000000] Thread (main_thread) [0x003367ec]} sys_vm: sys_vm_memory_map(vsize=0x2000000, psize=0xa00000, cid=0xffffffff, flags=0x200, policy=0x1, addr=*0xd001ddb0)
E {PPU[0x1000000] Thread (main_thread) [0x003367ec]} sys_vm: sys_vm_memory_map(vsize=0x2000000, psize=0x800000, cid=0xffffffff, flags=0x200, policy=0x1, addr=*0xd001ddb0)
E {PPU[0x1000000] Thread (main_thread) [0x003367ec]} sys_vm: sys_vm_memory_map(vsize=0x2000000, psize=0xd00000, cid=0xffffffff, flags=0x200, policy=0x1, addr=*0xd001ddb0)
E {PPU[0x1000000] Thread (main_thread) [0x003367ec]} sys_vm: sys_vm_memory_map(vsize=0x4000000, psize=0x2900000, cid=0xffffffff, flags=0x400, policy=0x1, addr=*0xd001ddd0)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=0, new_entry=*0xd0040a98)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=1, new_entry=*0xd0040a80)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=2, new_entry=*0xd0040a68)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=3, new_entry=*0xd0040a50)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=4, new_entry=*0xd0040a38)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=5, new_entry=*0xd0040a20)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=6, new_entry=*0xd0040a08)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=7, new_entry=*0xd00409f0)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=8, new_entry=*0xd00409d8)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=9, new_entry=*0xd00409c0)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=10, new_entry=*0xd00409a8)
U {PPU[0x1000000] Thread (main_thread) [0x02676958]} sysPrxForUser TODO: sys_crash_dump_set_user_log_area(index=11, new_entry=*0xd0040990)
U {PPU[0x1000000] Thread (main_thread) [0x003800e4]} sys_spu TODO: Unsupported SPU Thread Group type (0x18)
U {PPU[0x1000000] Thread (main_thread) [0x00380174]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
U {PPU[0x1000000] Thread (main_thread) [0x003800e4]} sys_spu TODO: Unsupported SPU Thread Group type (0x18)
U {PPU[0x1000000] Thread (main_thread) [0x00380174]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
U {PPU[0x1000000] Thread (main_thread) [0x003800e4]} sys_spu TODO: Unsupported SPU Thread Group type (0x18)
U {PPU[0x1000000] Thread (main_thread) [0x00380174]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
U {PPU[0x1000000] Thread (main_thread) [0x003800e4]} sys_spu TODO: Unsupported SPU Thread Group type (0x18)
U {PPU[0x1000000] Thread (main_thread) [0x00380174]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
U {PPU[0x1000000] Thread (main_thread) [0x003800e4]} sys_spu TODO: Unsupported SPU Thread Group type (0x18)
U {PPU[0x1000000] Thread (main_thread) [0x00380174]} sys_spu TODO: Unimplemented SPU Thread options (0x2)
E {PPU[0x1000004] Thread (null:) [0x02762090]} 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_bdvd/PS3_GAME/USRDIR/data/loc/locale.bh” [1]
```
The other version that I tried was 85920253b3fd9fc845057b2fa3972274e7b6607e. The game could be loaded, then I was able to enter the main menu, select my game mode. When the match started, the graphic froze but audio was running fine in the background. I saw continuous log being reproduced like below.

```
F {Worker 1} SPU: Trampoline simplified at 0x19814 (level=1) x4
F {Worker 4} SPU: Trampoline simplified at 0x197e0 (level=1) x4
F {Worker 3} SPU: Trampoline simplified at 0x19d90 (level=1) x4
F {Worker 0} SPU: Trampoline simplified at 0x199b8 (level=1) x4
F {Worker 7} SPU: Trampoline simplified at 0x1af50 (level=1) x4
F {Worker 5} SPU: Trampoline simplified at 0x1aa88 (level=1) x4
F {Worker 9} SPU: Trampoline simplified at 0x1aa74 (level=1) x4
F {Worker 11} SPU: Trampoline simplified at 0x1bfa8 (level=1) x4
F {Worker 2} SPU: Trampoline simplified at 0x1a9e0 (level=1) x4
F {Worker 10} SPU: Trampoline simplified at 0x1aa64 (level=1) x4
```
I tried different setting such as OpenGL (radeonsi) vs Vulkan (radv, amdvlk) but none of these settings help. 

System specification:
Ryzen 5 2600
16gb ram
Radeon RX580 with AMDGPU driver
Debian Sid running Linux 5.1.0

If you need more detail, please let me know.

**Comment by vsub** [comment]
Try clearing the SPU Cache
Recently there was done many changes to the spu and the old cache files doesn't seem to work very well

**Comment by e88z4** [comment]
For this commit https://github.com/RPCS3/rpcs3/commit/cc8c6358552f47627e7fda422141615880a71bd7
clearing SPU cache didn't help resolving the issue. 

The game is totally unplayable with this commit.
https://github.com/RPCS3/rpcs3/commit/85920253b3fd9fc845057b2fa3972274e7b6607e
Meaning, game can be loaded but as soon as the match starts, graphic froze and audio kept running.

**Comment by connorwalks** [comment]
I am also experiencing the same issue with the US version of the game. Previously it was perfectly playable

**Comment by RainbowCookie32** [comment]
If it started happening after #5944, then you should add to #5965 instead of opening a new issue. Also for future reference, you should reference the PR that broke it in the title instead of the commit. In my opinion it makes it more readable and easier to identify than a commit hash (especially a full one). 

And to wrap up, instead of pasting a piece of the log, upload the log file instead. You can post it here or upload it somewhere else if it's too big and add a link here. On Linux, the log is located in `~/.cache/rpcs3/`

**Comment by connorwalks** [comment]
Well I can confirm it is indeed #5944 that causes the issue. Will add logs for both to #5965

**Comment by AniLeo** [comment]
Closing as duplicate of #5965 
Please resume discussion there
